tests/sample_size/test_multiple_testing.py

```diff
@@ -96,6 +96,35 @@ def test_get_multiple_sample_size_does_not_converge(self, mock_expected_power):
             f"Couldn't find a sample size that satisfies the power you requested: {DEFAULT_POWER}",
         )
 
+    @parameterized.expand(
+        [
+            ("boolean", {"probability": 0.05, "mde": 0.02}, 1908),
+            ("numeric", {"variance": 5000, "mde": 5}, 3177),
+            (
+                "ratio",
+                {
+                    "numerator_mean": 500,
+                    "numerator_variance": 500000,
+                    "denominator_mean": 20,
+                    "denominator_variance": 2000,
+                    "covariance": 25000,
+                    "mde": 1,
+                },
+                20096,
+            ),
+        ],
+    )
+    def test_get_multiple_sample_size_fixed_output(self, metric_type, metadata, test_sample_size):
+        test_metric = {"metric_type": metric_type, "metric_metadata": metadata}
+
+        calculator = SampleSizeCalculator()
+        calculator.register_metrics([test_metric] * 2)
+
+        for s in range(1, 6):
+            np.random.seed(s)
+            sample_size = calculator.get_sample_size()
+            self.assertEqual(sample_size, test_sample_size)
```

Comment:
I might suggest something more like this?

```python3
# can be reused for other tests
TEST_BOOLEAN = {"metric_type": "boolean", "metric_metadata": {"probability": 0.05, "mde": 0.02}}
TEST_NUMERIC = {"metric_type": "numeric", "metric_metadata": {"variance": 5000, "mde": 5}}
TEST_RATIO = {
    "metric_type": "ratio",
    "metric_metadata": {
        "numerator_mean": 500,
        "numerator_variance": 500000,
        "denominator_mean": 20,
        "denominator_variance": 2000,
        "covariance": 25000,
        "mde": 1,
    },
}
...

    @parameterized.expand(
        (TEST_BOOLEAN, 1908, 1),
        (TEST_NUMERIC, 3177, 1),
        (TEST_RATIO, 20096, 1),
        (TEST_BOOLEAN, 1908, 1024),
        (TEST_NUMERIC, 3177, 1024),
        (TEST_RATIO, 20096, 1024),
    )
    def test_get_multiple_sample_size_fixed_output(self, test_metric, test_sample_size, seed):
        with patch("sample_size.metrics.multiple_testing.RANDOM_SEED", seed):
            calculator = SampleSizeCalculator()
            calculator.register_metrics([test_metric] * 2)
            sample_size = calculator.get_sample_size()
            self.assertEqual(sample_size, test_sample_size)
```

I've used seeds `1` and `1024` here, but what might be nice would be after `RANDOM_SEED` is refactored to a variable, you could set the test up to just print `metric_type, sample_size, seed`, and iterate through hundreds of seeds. You could go as far as only printing if the seed doesn't match `1908, 3177, 20096`. Something like

```python3
    @parameterized.expand(
        product((TEST_BOOLEAN, TEST_NUMERIC, TEST_RATIO), range(10 ** 3))
    )
    def test_dummy_search_for_seed(self, test_metric, seed):
        with patch("sample_size.metrics.multiple_testing.RANDOM_SEED", seed):
            calculator = SampleSizeCalculator()
            calculator.register_metrics([test_metric] * 2)
            sample_size = calculator.get_sample_size()
            if not sample_size in [1908, 3177, 20096]:
                 print(TEST_BOOLEAN['metric_type'], seed, sample_size)
            assert False
```

You could then use whatever seeds you find that produce alternative output